.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @arayaryoma
```

Comment:
Additionally, I would like to nominate @negibokken too to load balance review requests.

@negibokken are you okay?

Reply:
@tetsuharuohzeki Sure!

Reply:
@arayaryoma could you add their id?